fixed footer issue

sauravrajleaf opened the pull request:
## Issue
Fixes #75 

## Description
New content container add to Home & Bookmarked. Modified footer css.

## Type of change
- [x] Bug fix #75  (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change

## Screenshot
![Screenshot_Mozilla Firefox_50](https://user-images.githubusercontent.com/52833031/147685780-1746268e-55b7-440d-98ba-76c84417b848.png)
![Screenshot_Mozilla Firefox_51](https://user-images.githubusercontent.com/52833031/147685784-2553a202-d1d4-4795-88d1-c1c001ba45b8.png)
![Screenshot_Mozilla Firefox_52](https://user-images.githubusercontent.com/52833031/147685785-15c963f9-3d7c-4c16-b5fe-4ad0bce01905.png)
![Screenshot_Mozilla Firefox_53](https://user-images.githubusercontent.com/52833031/147685786-236fc89b-49a4-47d8-8040-743d52f8d5a4.png)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] The app runs properly
- [x] I have performed a self-review of my own code
- [x] I have formatted and commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

sauravrajleaf commented:
Please check if this can be merged.

Sparsh1212 reviewed (changes requested):
@sauravrajleaf Please remove all the editor formatting changes like double quotes, semicolons, etc.
A quicker way to do them would be to revert all the current changes and turn auto-format on save off. Now do the correct changes in CSS and then push them to your PR.
After this you are free to turn it on again :)

Sparsh1212 commented:
Please remove the merge conflicts too.

sauravrajleaf commented:
- As per the instructions, I have reverted all the changes done in all the files which was affected due to the code editor format on save. Later, I made changes to the required CSS. 
- Also regarding the merge conflict it was occurring in Home.js file which was also changed by the editor on save option as per my knowledge. So, in my opinion reverting everything would have solved that too.

sauravrajleaf commented:
Regarding the last part pushing them to my PR. I did not understand that very well. 

- Should I make a new PR & close this one?
- Should I commit the changes to same PR? 

If it's the later one, I have no idea to how to push changes to same PR. Can you please guide me through. Thanks.

Sparsh1212 commented:
> * As per the instructions, I have reverted all the changes done in all the files which was affected due to the code editor format on save. Later, I made changes to the required CSS.
> * Also regarding the merge conflict it was occurring in Home.js file which was also changed by the editor on save option as per my knowledge. So, in my opinion reverting everything would have solved that too.

Just a correction to your second point: The merge conflict is not due to the editor. It occurs when someone also proposed some changes on the same line of code that you were working and his/her changes got merged first before you. That's why you faced a merged conflict. To resolve this, I'd suggest you to watch any tutorial on "How to resolve merge conflicts"

Sparsh1212 commented:
> Regarding the last part pushing them to my PR. I did not understand that very well.
> 
> * Should I make a new PR & close this one?
> * Should I commit the changes to same PR?
> 
> If it's the later one, I have no idea to how to push changes to same PR. Can you please guide me through. Thanks.

By pushing I meant, just do a new commit to your current PR and push those latest changes. By pushing, your newly added commits will also reflect here, and I will be able to see your updated code.

sauravrajleaf commented:
I've done the changes. Please check once. I hope this time it's right.

Sparsh1212 reviewed (changes requested):
The web view is working totally fine. However, in the mobile view the UI is breaking.
Here take a look:
<img width="337" alt="Screenshot 2022-01-03 at 12 25 44 PM" src="https://user-images.githubusercontent.com/55937724/147905684-990e0e9e-ecbc-4fd8-ab16-1aa44ef65afe.png">
Please make sure it is fine in the mobile view too.

Also please revert the changes made to `package-lock.json` as those also have been made automatically by your ide.
